chat-plugins/playlist.js

```diff
@@ -0,0 +1,96 @@
+/*******************************
+*                              *
+*        Playlists             *
+*        By Execute            *
+*   Useful for sharing music.  *
+*                              *
+********************************/
+
+'use strict';
+
+let fs = require('fs');
+
+let blackcss = 'background:#3C3B42;padding:10px;color:white;border:1px solid black; border-radius:6px;text-align:center;';
+
+let data = {};
+
+try {
+	data = JSON.parse(fs.readFileSync('config/playlist.json', 'utf8'));
+} catch (e) {
+	console.log('Error reading playlist data, resorting to resetting all data entries....');
+	fs.writeFile('config/playlist.json', JSON.stringify(data));
+	console.log('Playlist data erased.');
+}
+
+function save() {
+	fs.writeFile('config/playlist.json', JSON.stringify(data));
+}
+
+function playlistGenerator(user) {
+	let playlistData = data[user];
+	if (!data[user] || data[user].length < 1) return Exiled.font('This user does not have any playlist data.', 'maroon', true);
+	let display = '<div style="' + blackcss + '">';
+	for (let i = 0; i < playlistData.length; i++) {
+		display += '<button name="send" value="/playlist play, ' + playlistData[i][0] + ', ' + user + '">' + playlistData[i][1] + '</button>';
+	}
+	display += '</div>';
+	return display;
+}
+
+exports.commands = {
+	playlist : function (target, room, user) {
+		if (!target) return this.sendReply('|uhtml|' + user.name + 'playlist|' + playlistGenerator(user.name));
+		let parts = target.split(',');
+		switch (toId(parts[0])) {
+		case 'add':
+			if (parts.length < 3) return this.parse('/playlist help');
+			let link = parts[1].trim();
+			let title = parts[2].trim();
+			if (!data[user.userid]) data[user.userid] = [];
+			if (data[user.userid].length > 9) return this.errorReply('You already exceded the maxium amount of songs a user may have in a playlist.');
+			for (let i = 0; i < data[user.userid].length; i++) {
+				if (data[user.userid][i][0] === link) return this.errorReply('You already have this song in your playlist.');
+				continue;
+			}
+			data[user.userid].push([link, title]);
+			save();
+			return this.sendReply('This song has been added to your playlist!');
+			break;
```

Comment:
unreachable code:

your `return`ing before it can reach the `break;` its safe to remove the break statement.

There are multiple of these in this file.